shared/utils/config.go

```diff
@@ -32,3 +36,21 @@ func LoadConfig(cfg interface{}) {
 
 	log.Debug("Config loaded: ", cfg)
 }
+
+func ParseMemoryLimit(limit string) (int64, error) {
+	unit := limit[len(limit)-2:]
```

Comment:
I think this will fail if `G` instead of `GB` is used, right?

Reply:
Fixed in the latest version